src/traces/scatter/calc.js

```diff
@@ -96,8 +96,9 @@ function calcAxisExpansion(gd, trace, xa, ya, x, y, ppad) {
         yOptions.padded = false;
     }
 
-    Axes.expand(xa, x, xOptions);
-    Axes.expand(ya, y, yOptions);
+    // N.B. asymmetric splom traces call this with undefined xa or ya
```

Comment:
not `undefined`, that would error here as well as above... it's [`{}`](https://github.com/plotly/plotly.js/blob/24b291285a15650a76ae6c78e2762a8e88f2027d/src/traces/splom/index.js#L66-L67)

Reply:
good catch -> 026cd53